### PR TITLE
app-shells/zsh-syntax-highlighting: add 0.8.0 and adoption

### PR DESCRIPTION
**Pull request by beatussum**

### CODE REVIEW

**Review by ceamac — COMMENTED** [review]
Please check the open bugs and add any needed `Closes:` tag(s) to the commit message.

File: app-shells/zsh-syntax-highlighting/zsh-syntax-highlighting-0.8.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	edo sed -i "s/COPYING.md//" Makefile
+}
+
+src_install() {
+	emake \
+		SHARE_DIR="${ED}/usr/share/zsh/site-functions" \
+		DOC_DIR="${ED}/usr/share/doc/${PF}" \
+		install
+
+	readme.gentoo_create_doc
```
[the remark this answers is not in the document]
Comment (beatussum) [reply app-shells/zsh-syntax-highlighting/zsh-syntax-highlighting-0.8.0.ebuild:43]: It was something already missing in `zsh-syntax-highlighting-0.7.1.ebuild`. I've fixed that. Thanks for review.

File: app-shells/zsh-syntax-highlighting/zsh-syntax-highlighting-0.8.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+DOC_CONTENTS="\
+In order to use ${CATEGORY}/${PN}, add
+\`source /usr/share/zsh/site-functions/zsh-syntax-highlighting.zsh\`
+at the end of your ~/.zshrc"
+
+src_prepare() {
+	default
+
+	edo sed -i "s/COPYING.md//" Makefile
```
[the remark this answers is not in the document]
Comment (beatussum) [reply app-shells/zsh-syntax-highlighting/zsh-syntax-highlighting-0.8.0.ebuild]: Okay, sorry I had not correctly understood the role of `edo`.

**Comment by beatussum** [comment]
> Please check the open bugs and add any needed `Closes:` tag(s) to the commit message.

I've linked this pull request to [920942](https://bugs.gentoo.org/920942).

**Review by ceamac — APPROVED** [review]
Thanks!

**Pull request closed without merging**